roles/openshift_facts/defaults/main.yml

```diff
@@ -10,11 +10,12 @@ l_openshift_images_dict:
   origin: 'docker.io/openshift/origin-${component}:${version}'
   openshift-enterprise: 'registry.redhat.io/openshift3/ose-${component}:${version}'
 
+l_osm_registry_url_default: "{{ l_openshift_images_dict[openshift_deployment_type] }}"
+l_oreg_host_temp: "{{ oreg_url | default(l_osm_registry_url_default) }}"
 # oreg_url is defined by user input.
-oreg_host: "{{ oreg_url.split('/')[0] if (oreg_url is defined and '.' in oreg_url.split('/')[0]) else '' }}"
+oreg_host: "{{ l_oreg_host_temp.split('/')[0] }}"
```

Comment:
Won't this fail if they've set oreg_url to a value that's not fully qualified, ex `oreg_url=repo/${component}:${version}`

Reply:
We've never allowed that, afaik.  Perhaps we should check for fqdn in oreg_url?

Reply:
Yeah, we probably should.

Reply:
I say that because we're actively advocating that people stop relying on the Red Hat specific --additional-registries flag for docker which means we should use fully qualified pull specs everywhere possible but make it easy for those to be targetted at other registries.